lib/msal-common/src/network/ThrottlingUtils.ts

```diff
@@ -94,12 +95,16 @@ export class ThrottlingUtils {
         ) * 1000);
     }
 
-    static removeThrottle(cacheManager: CacheManager, clientId: string, authority: string, scopes: Array<string>, homeAccountIdentifier?: string): boolean {
+    static removeThrottle(cacheManager: CacheManager, clientId: string, authority: string, scopes: Array<string>, homeAccountIdentifier?: string, request?: BaseAuthRequest): boolean {
```

Comment:
Can we make the request required, since the next common release will be breaking anyway?

Reply:
Will do.

Reply:
Done